src/registry/persistent.rs

```diff
@@ -15,6 +15,22 @@ pub struct PersistentRegistry {
     folder: PathBuf,
 }
 
+impl PersistentRegistry {
+    /// Creates a new [`PersistentRegistry`].
+    #[must_use]
+    pub fn new() -> Self {
+        Self::default()
+    }
+}
+
+impl Default for PersistentRegistry {
+    fn default() -> Self {
+        Self {
+            folder: PathBuf::from("./registry"),
```

Comment:
When creating a persistent repository, the folder in witch the data is stored should be the user choose, a default implementation is non-sense.